pom.xml

```diff
@@ -1911,6 +1911,7 @@
                 <param>
                   com.puppycrawl.tools.checkstyle.checks.naming.PatternVariableNameCheck*
```

Comment:
I don't think this check should be here too.

Reply:
https://github.com/checkstyle/checkstyle/pull/8649